build/manifest/main.go

```diff
@@ -102,20 +109,33 @@ func dumpPluginVersion(manifest *model.Manifest) {
 
 // applyManifest propagates the plugin_id into the server and webapp folders, as necessary
 func applyManifest(manifest *model.Manifest) error {
+	// generate JSON representation of Manifest.
+	// * JSON is very similar and compatible with JS's object literals. so, what we do here
+	//   is actually JS code generation.
+	// * this JSON ouput will also be used by the Go version to parse it internally to
+	//   expose a *model.Manifest.
+	manifestBytes, err := json.MarshalIndent(manifest, "", " ")
```

Comment:
Could we switch this to indent with two spaces, and match other `.json` files as used in our various repositories?
```suggestion
	manifestBytes, err := json.MarshalIndent(manifest, "", "  ")
```